docs/guides/represent-quantum-computers.ipynb

```diff
@@ -116,7 +113,7 @@
     "from qiskit.transpiler.preset_passmanagers import generate_preset_pass_manager\n",
     "\n",
     "pass_manager = generate_preset_pass_manager(\n",
-    "    optimization_level=1, target=target, seed_transpiler=12345\n",
+    "    target=target, seed_transpiler=12345\n",
```

Comment:
Was this intentional?

Reply:
yes, because we said it was the default value; so why include it?  But now I see that it's required (then why have a default value??) so I'm putting it back.